src/telegraf.ts

```diff
@@ -199,12 +205,10 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
 
   private botInfoCall?: Promise<tt.UserFromGetMe>
   async handleUpdate(update: tt.Update, webhookResponse?: http.ServerResponse) {
-    debug('Processing update', update.update_id)
```

Comment:
This has always been the most useful debug line for me. What's the rationale behind dropping it?

Reply:
No reason, I'll leave it in then. I think it should be moved until after `getMe` call tho, do you agree?

Reply:
Yes